Dockerfile

```
@@ -84,6 +84,17 @@ RUN set -eux; \
     curl -Lso /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_"$(dpkg --print-architecture)"; \
     chmod 755 /usr/local/bin/yq
 
+# pgbouncer-exporter
+ARG PGBOUNCER_EXPORTER_VERSION="0.7.0"
+RUN curl --silent \
+        --location \
+        --output /tmp/pkg.tgz \
+        https://github.com/prometheus-community/pgbouncer_exporter/releases/download/v${PGBOUNCER_EXPORTER_VERSION}/pgbouncer_exporter-${PGBOUNCER_EXPORTER_VERSION}.linux-amd64.tar.gz && \
```

As this repo may be building not for amd64, should we either make it dependent on the arch:

```suggestion
        https://github.com/prometheus-community/pgbouncer_exporter/releases/download/v${PGBOUNCER_EXPORTER_VERSION}/pgbouncer_exporter-$(dpkg --print-architecture).linux-${ARCH}.tar.gz && \
```

Or only build if we are building for amd64?